src/components/Loader.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Loader from "react-loader-spinner";
+
+const spin = () => {
```

Comment:
You  should write *spin* component in plural case

Reply:
I also think that the spin svg be at the center.